Src/MoneyFox.Core/Queries/Statistics/GetAccountProgression.cs

```diff
@@ -52,12 +52,18 @@ public Handler(IAppDbContext appDbContext, ISettingsFacade settingsFacade)
 
         public async Task<List<StatisticEntry>> Handle(Query request, CancellationToken cancellationToken)
         {
-            var payments = await appDbContext.Payments.Include(x => x.Category)
```

Comment:
I have the suspicion that the sum calculation does not work properly when you select all accounts in regards to transfers. When you look at a transfer on the level of all accounts then this should be a zero sum transaction, but I think right now it is added as income. Maybe you can extend your unit test and verify it that way?
(very nice btw that you added tests :) )

Reply:
You correctly pointed it out. The calculation is incorrect for the transfer payment. I have updated the logic now.